gh-pages merge

prabhasp opened the pull request.

strob commented:
Why are we still using this repo instead of monsooncollective/yatayat?
                                                - or -
Pull it yourself!

prabhasp committed:
fixes #5 by removing mangling orderedListofStops, but without deeper changes

Commit:
datasync: overpass files updated

Commit:
datasync: overpass files updated

suress committed:
fixes from Prabhas

Commit:
datasync: overpass files updated

Commit:
datasync: overpass files updated

Commit:
datasync: overpass files updated

Commit:
datasync: overpass files updated

Commit:
datasync: overpass files updated

Commit:
datasync: overpass files updated

Commit:
datasync: overpass files updated

Commit:
datasync: overpass files updated

Commit:
datasync: overpass files updated

Commit:
datasync: overpass files updated

Commit:
datasync: overpass files updated

Commit:
datasync: overpass files updated

Commit:
datasync: overpass files updated

Commit:
datasync: overpass files updated

Commit:
datasync: overpass files updated

Commit:
datasync: overpass files updated

Commit:
datasync: overpass files updated

Commit:
datasync: overpass files updated

neogeomat committed:
Documentation of YY.fromConfig

neogeomat committed:
Documentation of YY.fromOSM

Commit:
datasync: overpass files updated

Commit:
datasync: overpass files updated

Commit:
datasync: overpass files updated

Commit:
datasync: overpass files updated

Commit:
datasync: overpass files updated

Commit:
datasync: overpass files updated

neogeomat committed:
documentation of functions

neogeomat committed:
ready to push

neogeomat committed:
Merge branch 'gh-pages' of https://github.com/monsooncollective/yatayat into gh-pages

Conflicts:
	yatayat.js

neogeomat committed:
Merge branch 'gh-pages-samar' into gh-pages

Conflicts:
	index.html

neogeomat committed:
Merge branch 'ordering_with_kdtree' into gh-pages

Conflicts:
	yatayat.js

neogeomat committed:
small changes

neogeomat committed:
Merge branch 'understanding_routing' into gh-pages

Conflicts:
	index.html

Commit:
datasync: overpass files updated

prabhasp committed:
Merge branch 'gh-pages' of github.com:monsooncollective/yatayat into neogeomat-gh-pages

prabhasp committed:
merging @suress and @neogeomat's changes

prabhasp committed:
css and language revisions

prabhasp committed:
removing the geo-coder; more appropriate on the right sidebar

prabhasp committed:
Merge pull request #8 from monsooncollective/neogeomat-gh-pages

Merging changes from Suresh and Amrit's work.

Commit:
datasync: overpass files updated

prabhasp committed:
fixing typos / minor errors

prabhasp committed:
Merge branch 'neogeomat-gh-pages' into gh-pages

prabhasp committed:
linking back to yy.moncol

prabhasp committed:
minor fix

prabhasp committed:
getting rid of z() things

Commit:
datasync: overpass files updated

Commit:
datasync: overpass files updated

Commit:
datasync: overpass files updated

Commit:
datasync: overpass files updated

Commit:
datasync: overpass files updated

Commit:
datasync: overpass files updated

Commit:
datasync: overpass files updated

Commit:
datasync: overpass files updated

neogeomat committed:
Merge pull request #13 from monsooncollective/gh-pages

merged changes.

Commit:
datasync: overpass files updated

Commit:
datasync: overpass files updated

Commit:
datasync: overpass files updated

Commit:
Commit from linux

neogeomat committed:
Merge branch 'gh-pages' of github.com:amritkarma/yatayat into gh-pages

neogeomat committed:
same color in instruction and route done, need to correct the layout

Commit:
datasync: overpass files updated

Commit:
datasync: overpass files updated

Commit:
datasync: overpass files updated

Commit:
datasync: overpass files updated

Commit:
datasync: overpass files updated

Commit:
datasync: overpass files updated

neogeomat committed:
Use Same color coding in instruction and route display completed.

neogeomat committed:
removed extra data files

neogeomat committed:
Merge remote-tracking branch 'remotes/monsooncollective/gh-pages' into gh-pages

neogeomat committed:
added CNAME to ignore list and changed the osm file to latest

neogeomat committed:
updated leaflet.js and add leaflet/leaflet-src.js

Commit:
datasync: overpass files updated

neogeomat committed:
added stoptosegdict in system

neogeomat committed:
Merge branch 'gh-pages' into removing_overshoots

neogeomat committed:
added CNAME again

neogeomat committed:
tile layer working again

neogeomat committed:
Merge branch 'removing_overshoots' into gh-pages

Commit:
datasync: overpass files updated

Commit:
datasync: overpass files updated

Poshan committed:
autocompletion fast with jquery-ui

Commit:
datasync: overpass files updated

Poshan committed:
few edits

neogeomat committed:
Merge branch 'gh-pages' into autocomplete-fast

Conflicts:
	CNAME
	config.stable.json
	index.html
	lib/leaflet/Control.OSMGeocoder.css
	lib/leaflet/Control.OSMGeocoder.js
	lib/leaflet/leaflet.js
	lib/routeinfopanel.js
	lib/yatayat.css
	transit.experimental.xml
	yatayat.js

Poshan committed:
ALphabetical ordering of routes in routeinfopanel complete

neogeomat committed:
merging test

neogeomat committed:
merged changes from @Poshan

neogeomat committed:
Merge branch 'alphabetical-ordering-in-routeinfopanel' into gh-pages

Conflicts:
	index.html
	lib/routeinfopanel.js

neogeomat committed:
testing alphabetical ordering from Poshan

neogeomat committed:
Merge branch 'alphabetical-ordering-in-routeinfopanel' into gh-pages

Conflicts:
	lib/routeinfopanel.js

neogeomat committed:
need to add control on click

neogeomat committed:
buttons working ready to push. solved few issues like faster autocompletion in stop selection and alphabetical ordering of routes in routeinfopanel.js

Signed-off-by: amritkarma <amrit.im@gmail.com>

neogeomat committed:
Merge branch 'gh-pages' of https://github.com/monsooncollective/yatayat into gh-pages

Conflicts:
	transit.experimental.xml

Commit:
datasync: overpass files updated

Commit:
datasync: overpass files updated

Commit:
datasync: overpass files updated

neogeomat committed:
added time measure for system object creation

neogeomat committed:
added timeing to measure performance

neogeomat committed:
removed older google suggestion form

Commit:
datasync: overpass files updated

neogeomat committed:
Merge branch 'gh-pages' of https://github.com/monsooncollective/yatayat into gh-pages

Commit:
datasync: overpass files updated

Commit:
datasync: overpass files updated

neogeomat committed:
removed unneccessary console.log

neogeomat committed:
bigreko nilaiyo, leaflet changed to old version

neogeomat committed:
bigreko nilaiyo, leaflet changed to old version

Commit:
datasync: overpass files updated

neogeomat committed:
finally something is working, not as expected but still ......

neogeomat committed:
actual code where inserting changes for stop not working, matching coordinates not working, need to reference by id,

neogeomat committed:
add dict of node and list of node ids in segment object

neogeomat committed:
overshoots completed, but still some other error like retarded response

neogeomat committed:
finished overshoots

neogeomat committed:
Merge branch 'gh-pages' of https://github.com/monsooncollective/yatayat into removing_overshoots

neogeomat committed:
Merge branch 'gh-pages' of github.com:neogeomat/yatayat into removing_overshoots

Commit:
datasync: overpass files updated

neogeomat committed:
error found, autocomplete select should be called before typeahead.change function but it is happening otherewise.

neogeomat committed:
reverted o typeahead

Commit:
datasync: overpass files updated

neogeomat committed:
the problem is in route order method

Poshan committed:
bounds extend

Poshan committed:
Merge branch 'overshot' of https://github.com/neogeomat/yatayat into gh-pages

Poshan committed:
bounds extension for each segment completed

neogeomat committed:
Merge branch 'overshot_stop_knows_its_segment' into gh-pages

neogeomat committed:
problem of incorrect identification of segment when stops falls in more than one segment (like balaju stop is in 3 segments)

Poshan committed:
Button in routeinfopanel associated with correct routes

Commit:
datasync: overpass files updated

Commit:
datasync: overpass files updated

Commit:
datasync: overpass files updated

Commit:
datasync: overpass files updated

neogeomat committed:
added wiki, overshoot is postponed.

Commit:
datasync: overpass files updated

Commit:
datasync: overpass files updated

Poshan committed:
silly changes

neogeomat committed:
Merge remote-tracking branch 'remotes/monsooncollective/gh-pages' into gh-pages

neogeomat committed:
improved typeahead, redefined bounds of routed route. Associated correct route in routeinfopanel.js

Merge remote-tracking branch 'remotes/Poshan/gh-pages' into gh-pages

Conflicts:
	index.html

Poshan committed:
from pasadai

Poshan committed:
extend in single route

neogeomat committed:
consistent coloring in order of routed route, need to fix the colors

neogeomat committed:
consistent coloring in order of routed route, need to fix the colors

Commit:
datasync: overpass files updated

Commit:
datasync: overpass files updated

Commit:
datasync: overpass files updated

neogeomat committed:
color chosen for consistent coloring of routed route

neogeomat committed:
color chosen for consistent coloring of routed route

neogeomat committed:
Merge remote-tracking branch 'remotes/monsooncollective/gh-pages' into gh-pages

Commit:
datasync: overpass files updated

Commit:
datasync: overpass files updated

Commit:
datasync: overpass files updated

Commit:
datasync: overpass files updated

neogeomat committed:
Create gh-pages branch via GitHub

neogeomat committed:
Update CNAME

neogeomat committed:
Merge branch 'gh-pages' into ktmlabs-gh-pages

Conflicts:
	index.html

neogeomat committed:
Merge branch 'gh-pages' of https://github.com/KathmanduLivingLabs/yatayat into ktmlabs-gh-pages

neogeomat committed:
unidirectional testing

neogeomat committed:
added role to segments

neogeomat committed:
added arrow to give direction

neogeomat committed:
Merge branch 'unidirectional' into direction_of_segment

Conflicts:
	index.html

Commit:
datasync: overpass files updated

Commit:
datasync: overpass files updated

neogeomat committed:
Merge remote-tracking branch 'remotes/monsooncollective/gh-pages' into gh-pages

Commit:
datasync: overpass files updated

Poshan committed:
few changes

Poshan committed:
GOandSwap

Poshan committed:
merged from pasadai

Poshan committed:
merged with origin

Poshan committed:
few edits

neogeomat committed:
changed tile source to mapquest

neogeomat committed:
Merge branch 'monsoon-gh-pages' into gh-pages

neogeomat committed:
added go swap button, changed tile source

Commit:
datasync: overpass files updated

neogeomat committed:
commit to push

neogeomat committed:
Merge remote-tracking branch 'remotes/monsooncollective/gh-pages' into gh-pages

neogeomat committed:
Merge branch 'gh-pages' into direction_of_segment

Conflicts:
	yatayat.js

Commit:
datasync: overpass files updated

neogeomat committed:
added

neogeomat committed:
direction based on order of stop and not on based on segments.

neogeomat committed:
almost done, need internet connection

Commit:
datasync: overpass files updated

neogeomat committed:
atleast working

Commit:
datasync: overpass files updated

Commit:
datasync: overpass files updated

neogeomat committed:
done for atleast one track

neogeomat committed:
Merge remote-tracking branch 'remotes/monsooncollective/gh-pages' into direction_of_route_from_stop

neogeomat committed:
Merge branch 'direction_of_route_from_stop' into gh-pages

Conflicts:
	index.html

Commit:
datasync: overpass files updated

neogeomat committed:
changes to oveshoot

neogeomat committed:
Merge branch 'direction_of_route_from_stop' into gh-pages

Conflicts:
	index.html

neogeomat committed:
small changes

neogeomat committed:
added direction arrow to track

neogeomat committed:
start to branch for endsegments

neogeomat committed:
removed arrows on clicking clear

Commit:
datasync: overpass files updated

Commit:
datasync: overpass files updated

Commit:
datasync: overpass files updated

neogeomat committed:
Stop for now

neogeomat committed:
no idea

neogeomat committed:
Merge branch 'gh-pages' into ktmlabs_gh-pages

Commit:
datasync: overpass files updated

Commit:
datasync: overpass files updated

Commit:
datasync: overpass files updated

Commit:
datasync: overpass files updated

Commit:
datasync: overpass files updated

Commit:
datasync: overpass files updated

Commit:
datasync: overpass files updated

Commit:
datasync: overpass files updated

Commit:
datasync: overpass files updated

Commit:
datasync: overpass files updated

Commit:
datasync: overpass files updated

Commit:
datasync: overpass files updated

Commit:
datasync: overpass files updated

Commit:
datasync: overpass files updated

Commit:
datasync: overpass files updated

Commit:
datasync: overpass files updated

Commit:
datasync: overpass files updated

Commit:
datasync: overpass files updated

Commit:
datasync: overpass files updated

Commit:
datasync: overpass files updated

Commit:
datasync: overpass files updated

Commit:
datasync: overpass files updated

Commit:
datasync: overpass files updated

Commit:
datasync: overpass files updated

Commit:
datasync: overpass files updated

Commit:
datasync: overpass files updated

Commit:
datasync: overpass files updated

Commit:
datasync: overpass files updated

Commit:
datasync: overpass files updated

Commit:
datasync: overpass files updated

Commit:
datasync: overpass files updated

Commit:
datasync: overpass files updated

Commit:
datasync: overpass files updated

Commit:
datasync: overpass files updated

Commit:
datasync: overpass files updated

Commit:
datasync: overpass files updated

Commit:
datasync: overpass files updated

Commit:
datasync: overpass files updated

Commit:
datasync: overpass files updated

Commit:
datasync: overpass files updated

Commit:
datasync: overpass files updated

Commit:
datasync: overpass files updated

Commit:
datasync: overpass files updated

prabhasp committed:
Merge branch 'gh-pages' of https://github.com/KathmanduLivingLabs/yatayat into KathmanduLivingLabs-gh-pages

Conflicts:
	transit.experimental.xml

prabhasp committed:
Updating subproject to a valid commit

prabhasp committed:
kdtree updates

prabhasp committed:
kdtree in repo rather than as submodule

prabhasp committed:
whoops

prabhasp committed:
first commit

prabhasp committed:
Adding back route name popup
